solidity/test/BondedECDSAKeepFactoryTest.js

```diff
@@ -1140,7 +1140,7 @@ contract("BondedECDSAKeepFactory", async (accounts) => {
     }
   })
 
-  describe("setGroupSelectionSeed", async () => {
+  describe("__beaconCallback", async () => {
     const newGroupSelectionSeed = new BN(2345675)
```

Comment:
`newGroupSelectionSeed` -> `newRelayEntry`? We updated naming in https://github.com/keep-network/keep-ecdsa/pull/339/files#diff-5325d7067cb217b70e5bed5c308051faR281 so it would be worth aligning it here as well.

Reply:
Yep! See https://github.com/keep-network/keep-ecdsa/pull/339/commits/c9407c6aa3931426669046f879051e5b7b591f08